Documentation/ApiOverview/Localization/TranslationServer/Crowdin.rst

```diff
@@ -1,27 +1,31 @@
-.. include:: /Includes.rst.txt
-.. index::
-   Localization; Crowdin
-   Crowdin
-.. _xliff-translating-server-crowdin:
+..  include:: /Includes.rst.txt
+..  index::
+    Localization; Crowdin
+    Crowdin
+..  _xliff-translating-server-crowdin:
 
 =========================
 Localization with Crowdin
 =========================
 
-.. tip::
-
-   Crowdin is used since TYPO3 v10 by default.
-
 What is Crowdin?
 ================
 
-Crowdin is a localization management platform and offers the Core features essential for delivering great translation:
+`Crowdin`_ is a localization management platform and offers features essential
+for delivering great translation:
+
+*   **Single source**: translate text once that is used in different versions
```

Comment:
If you are reformating this anyway I would turn it into a definition list which has more semantic meaning